src/app/components/MnemonicValidation/index.tsx

```diff
@@ -80,7 +80,6 @@ export function MnemonicValidation(props: Props) {
               type="submit"
               label={t('openWallet.mnemonic.import', 'Import my wallet')}
               primary
-              onClick={onSubmit}
```

Comment:
the linter wants us to collapse this into one line after the removal. @alexander-elgin would you be able to make that change for us too?

Reply:
@pro-wh Done. Sorry, I did not check it at the first time